app/javascript/chat/chat.jsx

```diff
@@ -1341,6 +1329,45 @@ export default class Chat extends Component {
                 ) : null}
               </span>
             </button>
+            {this.state.isTagModerator ? (
+              <button
```

Comment:
refactor (blocking): Use the design system [Button](https://storybook.forem.com/?path=/story/3-components-buttons--default) component.